docs/self-managed/concepts/access-control/resource-authorizations.md

```diff
@@ -0,0 +1,20 @@
+---
+id: resource-authorizations
+title: "Resource authorizations"
+sidebar_label: "Resource authorizations"
+description: "Resource authorizations allow you to control the level of access a user, or group, has to a particular resource in the system."
+---
+
+:::caution
+Resource authorizations are enabled by the use of a feature flag. To enable this feature in Identity, see the
+[configuration variables feature flags](../../../../self-managed/identity/deployment/configuration-variables/#feature-flags) section.
```

Comment:
Since resource authorizations are not limited to Identity, we should also link to configuration guides in Operate and Tasklist. @marcosgvieira @johanwelgemoed can you provide the links we can use to the respective sections in your docs?

Reply:
CC @aabouzaid: When this features flag is added as a configuration value in the helm charts, a link should also be added here, so it's easier discoverable for users.

Reply:
In Operate following URL will be merged within [this PR](https://github.com/camunda/camunda-platform-docs/pull/1886): https://docs.camunda.io/docs/self-managed/operate-deployment/operate-authentication/#resource-based-permissions

Reply:
In Tasklist: 

- PR: https://github.com/camunda/camunda-platform-docs/pull/1887
- Page: https://docs.camunda.io/docs/self-managed/tasklist-deployment/tasklist-authentication/#resource-based-permissions